pdata/pcommon/map.go

```diff
@@ -216,6 +218,8 @@ func (m Map) Len() int {
 //	sm.Range(func(k string, v Value) bool {
 //	    ...
 //	})
+//
+// Deprecated: Use the iterator returned by All instead.
```

Comment:
Deprecation messages should have the version that the symbol was deprecated on

Reply:
Updated, thanks.